pages/flow/index.js

```diff
@@ -1,35 +1,39 @@
 import styled from "styled-components";
 import AsanaCard from "../../components/Asana-Card/AsanaCard";
-import { getAsanaByID } from "../../db";
 import asanas from "../../db";
 import { useState } from "react";
 import { MainButton } from "../../components/MainButton";
 
 export default function Flow() {
-  const [flowAsanaIds, setFlowAsanaIds] = useState([]);
+  const [selectedAsanas, setSelectedAsanas] = useState([]);
   const [open, setOpen] = useState(false);
 
+  function deleteAsana(cardIndex) {
```

Comment:
Your delete function should consume the id to identify the card instead of the index in the current array. Using the index might lead to bugs down the road (for example as soon as you implement a filter functionality for the user)

Reply:
Hi @ThomasF85, I used the index, because the user can add the same card several times to the array. When I use the id, then I would delete all cards with the same id at once from the array. But I could give the cards a new id (something like "list-id") when adding them to the state, and use this one for the delete function? Does it make sense?

Reply:
I see, in that case you can use the index. It's your choice whether you want to add another new id to the cards here or not. Just be aware that it might lead to bugs lateron if you don't.

Reply:
Ok, thanks for your feedback! I added a new id to the cards.